src/styles/index.scss

```diff
@@ -1,3 +1,63 @@
+$column-width: 100px;
+$height: 100px;
+$gap: 1px;
+$padding: 10px;
+$week-days: 0 'mon', 1 'tue', 2 'wed', 3 'thu', 4 'fri', 5 'sat', 6 'sun';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar{
```

Comment:
According to BEM rules, each BEM block should be in a separate file - move calendar styles to a separate file

Reply:
not fixed
<img width="799" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/98764641/a99e36a8-9020-47a8-95a1-81ac27677dc1">